modules/extension/wmts/src/test/java/org/geotools/ows/wmts/online/WMTSServerTest.java

```diff
@@ -47,15 +50,31 @@
 import org.opengis.referencing.crs.CoordinateReferenceSystem;
 import org.opengis.referencing.operation.TransformException;
 
-public class WMTSServerTest extends WMTSOnlineTestCase {
+public class WMTSServerTest extends OnlineTestCase {
 
     private URL server;
 
     @Before
-    public void setUp() throws MalformedURLException {
+    public void setUpInternal() throws MalformedURLException {
         this.server =
                 new URL(
-                        "https://nvdbcache.geodataonline.no/arcgis/rest/services/Trafikkportalen/GeocacheTrafikkJPG/MapServer/WMTS/1.0.0/WMTSCapabilities.xml");
+                        "https://api.lantmateriet.se/open/topowebb-ccby/v1/wmts/token/8d61b10d-e93b-3c04-b4ae-4f4bdd1afe1b/?request=getcapabilities&service=wmts");
+    }
+
+    @Override
+    protected Properties createExampleFixture() {
+        Properties example = new Properties();
```

Comment:
Ok so... the test is really checking for the presence of the property file?

Reply:
thanks @aaime, I put an exclusion tag for online test in pom.